.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "2017-2018/gameEngine/robomodules"]
+	path = 2017-2018/gameEngine/robomodules
+	url = git@github.com:HarvardURC/robomodules.git
```

Comment:
I typically try to use the HTTPS url when cloning a submodule so that if someone clones the root repository via HTTPS it won't break. I believe you can edit this file directly. 

Also might want to add a note to root project readmes that those projects need to be cloned recursively, since I think that isn't the default for some reason.

Reply:
can you fix this please?